resources/views/components/layouts/app/sidebar.blade.php

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="{{ str_replace('_', '-', app()->getLocale()) }}" class="dark">
     <head>
-        @include('partials.head')
+        @include('partials.head', ['title' => $title ?? null])
```

Comment:
```suggestion
        @include('partials.head')
```

You don't need to pass variables to `@include` as they are always available in the partial.

Reply:
thank you @Jubeki 👍